app/views/public/api_documentation.html.erb

```diff
@@ -409,6 +410,43 @@ curl -X POST -H "Token: secret-api-token" -H "Content-Type: application/json" -d
 
 <hr />
 
+<h4 id="sol-prices">Sol Prices</h4>
+<p>The Sol Prices endpoind will return prices gathered from multiple exchanges.</p>
+<p>Request:</p>
+<pre>curl -H "Token: secret-api-token" 'https://www.validators.app/api/v1/sol-prices'</pre>
+<p>Parameter Options:</p>
+<p>`from=%year-%month-%dayT%hour:%minute:%second` returns records starting from a given time - defaults to 30 days ago. </p>
```

Comment:
`will return results starting from a given time, defaults to 30 days.`
Please keep wording consistent.